taoswswrap/tdengine_connection.py

```diff
@@ -73,10 +103,17 @@ def values_to_column(values: list, column_type: str):
 
 
 class Statement:
-    def __init__(self, columns: dict[str, str], subtable: str, values: dict[str, Any]):
+    def __init__(
+        self,
+        columns: dict[str, str],
+        subtable: str,
+        values: dict[str, Any],
+        timestamp_precision: str = _TimestampPrecision.ms,
```

Comment:
Pretty sure this is supposed to be:
```suggestion
        timestamp_precision: str = _TimestampPrecision.ms.value,
```
or otherwise make `TimestampPrecision` public and a part of the API and change the type hint accordingly.

Reply:
Note that since it's a `_StrEnum`, the following holds:
```py
assert isinstance(_TimestampPrecision.ms, str)
assert _TimestampPrecision.ms == _TimestampPrecision.ms.value
```
But I'll change this enum to public